pulumi/python/kubernetes/nginx/ingress-controller-repo-only/__main__.py

```diff
@@ -31,7 +31,7 @@
     nginx_repository = "nginx/nginx-ingress"
 nginx_tag = config.get('nginx_tag')
 if not nginx_tag:
-    nginx_tag = "2.2.0"
+    nginx_tag = "2.4.0"
```

Comment:
What does this version number correlate to? I don't see it in the version numbers modified in the `.example` above.

Reply:
There are two paths for the NGINX IC deployment; one is in the mainline where we do everything in Python - this is the "just pull from the repo w/ the JWT" logic, and for that we need the nginx image tag (latest is 2.4.0). If you look at the NGINX version that's the most recent.

Reply:
Thanks for the clarification.  Ideally this lives in config somewhere but let's put that on https://github.com/nginxinc/kic-reference-architectures/issues/140